### PR TITLE
Utilize ignored suffixes in homepage search 

### LINKED ISSUE #1112: Failure of proposal search field on main page for 2593
When entering 2593 in the search box on the main page, the result that comes back is an error page that says "xxxx_pre-image.fits" does not follow jwst naming conventions.

Looks like this may be a new case that needs to be added to the filename parser.

### PR DESCRIPTION
**Pull request by BradleySappington**
Fix bug where some proposals dont load because they have `pre-image` suffix

### CODE REVIEW

**Comment by BradleySappington** [comment]
@mfixstsci @bhilbert4 - Ready for review!

File: jwql/website/apps/jwql/forms.py [hunk 1]
```diff
@@ -282,8 +272,8 @@ def clean_search(self):
             all_files = glob.glob(search_string_public)
             all_files.extend(glob.glob(search_string_proprietary))
 
-            # Ignore "original" files
-            all_files = [filename for filename in all_files if 'original' not in filename]
+            # Gather all files that do not have the 'IGNORED_SUFFIXES' in them
+            all_files = [filename for filename in all_files if not any(name in filename for name in IGNORED_SUFFIXES)]
```
Comment (bhilbert4) [inline jwql/website/apps/jwql/forms.py:276]: Wow. Impressive that you found a way to do this in a single list comprehension.

File: jwql/website/apps/jwql/forms.py [hunk 2]
```diff
@@ -58,20 +58,10 @@ def view_function(request):
...
-from jwql.utils.constants import FILTERS_PER_INSTRUMENT
-from jwql.utils.constants import GENERIC_SUFFIX_TYPES
-from jwql.utils.constants import GRATING_PER_INSTRUMENT
-from jwql.utils.constants import GUIDER_FILENAME_TYPE
-from jwql.utils.constants import JWST_INSTRUMENT_NAMES_MIXEDCASE
-from jwql.utils.constants import JWST_INSTRUMENT_NAMES_SHORTHAND
-from jwql.utils.constants import READPATT_PER_INSTRUMENT
-from jwql.utils.utils import get_config, get_rootnames_for_instrument_proposal, filename_parser
-from jwql.utils.utils import query_format
+from jwql.utils.constants import (ANOMALY_CHOICES_PER_INSTRUMENT, ANOMALIES_PER_INSTRUMENT, APERTURES_PER_INSTRUMENT, DETECTOR_PER_INSTRUMENT, EXP_TYPE_PER_INSTRUMENT,
```
Comment (bhilbert4) [inline jwql/website/apps/jwql/forms.py:61]: I'm always disappointed that PEP8 doesn't request an indentation for cases like this.

**Review by bhilbert4 — APPROVED** [review]
This looks good to me. Tested on the dev server and it seems to work well.

**Pull request merged**